apps/webapp/src/components/staking/account/header/index.tsx

```diff
@@ -45,33 +51,41 @@ export const Header = () => {
         <div className='flex flex-col gap-2'>
           <AccountSwitcher account={account} onChange={setAccount} />
 
-          <div className='flex justify-center gap-8'>
+          <div className='flex items-start justify-center gap-8'>
             <Stat label='Available to delegate'>
               <ValueViewComponent view={unstakedTokens ?? zeroBalanceUm} />
             </Stat>
 
             <Stat label='Unbonding amount'>
-              <div className='flex gap-2'>
-                <ValueViewComponent view={unbondingTokens?.total ?? zeroBalanceUm} />
-                <TooltipProvider>
-                  <Tooltip>
-                    <TooltipTrigger>
-                      <img src='./info-icon.svg' className='size-4' alt='An info icon' />
-                    </TooltipTrigger>
-                    <TooltipContent>
-                      <div className='flex flex-col gap-4'>
-                        <div className='max-w-[250px]'>
-                          Total amount of UM you will receive when all your unbonding tokens are
-                          claimed, assuming no slashing.
-                        </div>
-                        {unbondingTokens?.tokens.map(token => (
-                          <ValueViewComponent key={getDisplayDenomFromView(token)} view={token} />
-                        ))}
+              <TooltipProvider>
```

Comment:
All I did here was make the entire unbonding amount `ValueViewComponent` the tooltip trigger, rather than having the little (i) icon as the trigger